core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('Ck editor copy content service', () => {
     expect(service.copyModeActive).toBe(false);
     service.toggleCopyMode();
     expect(service.copyModeActive).toBe(true);
-
+    /* eslint-disable oppia/no-multiline-disable */
```

Comment:
For every place you use this, please leave a comment explaining why it is needed. If possible, don't use it at all, and have per-line disables instead.